src/components/TimeCardPage/SubmitCard.js

```diff
@@ -6,15 +6,27 @@ import Button from 'react-bootstrap/Button';
 
 
 
+
 export default function SubmitCard(props) {
 
     const [submitted, setSubmitted] = useState(false); 
     const [submitDate, setSubmitDate] = useState(null); 
+    const [state, setState] = useState('Rejected'); 
```

Comment:
🔧 What type is `state`? A data definition would be useful since we're using this as enumeration down below:

i.e.

```typescript
enum CardState {
  Rejected
  InReviewEmployer
  InReviewBreaktime
  Completed
}

```

Reply:
```typescript
const [state, setState] = useState<CardState>(CardState.Rejected); 
```